### DISCUSSION TITLE
Rust Server Console not working? Players spawning under map?

### DISCUSSION THREAD

**Discussion by Nastra89** [GameServerManagers/LinuxGSM#4442] [Q&A]
Hello i installed Linuxgsm on a linux mint 21.03 machine. i have to tell you that i´m new to linux.
I can run a rust server just fine but i have two issues.

First if i type ./rustserver console it switches to the console but it stuck there. With the RustAdmin everthing works fine!

![Lgsm server 1](https://github.com/GameServerManagers/LinuxGSM/assets/156541414/22164736-5c87-49b1-a7fd-837c6fc4662f)

It doesn´t matter if i put "verbost output on yes and interactive on no"

Thats the output:
![lgsm server 2](https://github.com/GameServerManagers/LinuxGSM/assets/156541414/4f32578d-4bd5-4395-b351-330c2ef851a6)

and there it stays forever.

my second problem is if i try to join my server its kicks me out with the message "InsideTerrain Violation Level 200" if i type "terrain_protection 0" in the rustadmin console i spawn under the map (for testing its only a 1500 size.
is there any way to solve the problem?

Sorry for my bad english.

Thanks and cheers!

**Comment by csplkiller** [comment]
I have the same problem.
Console not working.
LinuxGSM - Rust - Version v24.3.2
